resources/views/categories/edit.blade.php

```diff
@@ -15,11 +15,16 @@
 <div class="form-group {{ $errors->has('category_type') ? ' has-error' : '' }}">
     <label for="category_type" class="col-md-3 control-label">{{ trans('general.type') }}</label>
     <div class="col-md-7 required">
-        {{ Form::select('category_type', $category_types , old('category_type', $item->category_type), array('class'=>'select2', 'style'=>'min-width:350px', 'aria-label'=>'category_type', $item->itemCount() > 0 ? 'disabled' : '')) }}
+        {{ Form::select('category_type', $category_types , old('category_type', $item->category_type), array('class'=>'select2', 'style'=>'min-width:350px', 'aria-label'=>'category_type', ($item->category_type!='') || ($item->itemCount() > 0) ? 'disabled' : '')) }}
```

Comment:
I know it was already here before, but an array that looks like this seems weird to me:

```php
[
   'class' => 'select2',
   'style' => 'min-width:350px',
   'aria-label' => 'category_type',
   'disabled'
]
```

Should it maybe be `'disabled' => /* conditional thingee */`?

Reply:
That's how it's supposed to be